src/App.tsx

```diff
@@ -3,9 +3,74 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { MovieData } from './types/MovieData';
+import { ResponseError } from './types/ReponseError';
+
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [canAddMovie, setCanAddMovie] = useState(false);
```

Comment:
Can we avoid using this variable and depends on `selectedMovie` and `isLoading`?